k8s/k8s_integration_test.go

```diff
@@ -53,26 +54,26 @@ func (k *k8sSecretTest) TestPutSecret(t *testing.T) error {
 	k.passphrase = uuid.New()
 	secretData[secretId] = k.passphrase
 	// PutSecret with non-nil secretData and no namespace should fail
-	err := k.s.PutSecret(secretName, secretData, nil)
+	_, err := k.s.PutSecret(secretName, secretData, nil)
```

Comment:
is there a reason we are not testing the secret version for k8s?

Reply:
Fixed the k8s integration tests